internal/nginx-meshctl/commands/remove.go

```diff
@@ -41,6 +42,10 @@ const exampleRemove = `
 
 var yes bool
 
+// proxiedResources is a map of namespace -> k8s resource type -> resource names;
```

Comment:
Nit: `{"namespace": {"resource type": ["resource names"]}}` makes it a bit easier to visualize what this map will contain.